M/MiniZinc/build_tarballs.jl

```diff
@@ -48,7 +48,7 @@ dependencies = [
     Dependency("CompilerSupportLibraries_jll"),
     # Use an exact version for HiGHS. @odow has observed segfaults with
     # HiGHS_jll v1.5.3 when libminizinc compiled with v1.5.1.
-    Dependency("HiGHS_jll"; compat="=1.7.1"),
+    Dependency("HiGHS_jll"; compat="=1.11.0"),
```

Comment:
Does this still need to be a hard version constraint? The comment makes it sound like it was an issue with a specific older release of HiGHS, so can this be relaxed now?

Reply:
Not sure. HiGHS has made a few accidental breaking changes to their C API between releases (adding new fields to the middle of structs, etc). It's safer just to lock?